packages/tasks/src/model-libraries.ts

```diff
@@ -275,6 +275,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoName: "mindspore",
 		repoUrl: "https://github.com/mindspore-ai/mindspore",
 	},
+	"mars5-tts": {
+		prettyLabel: "MARS5-TTS",
+		repoName: "MARS5-TTS",
+		repoUrl: "https://github.com/Camb-ai/MARS5-TTS",
+		filter: false,
+		countDownloads: `path_filename:"mars5_ar.safetensors"` },
```

Comment:
Cannot be merged yet! Since it uses the new searchquery by @Kakulukian

Reply:
`path_filename` doesn't contain the extension, see https://github.com/huggingface/huggingface.js/pull/749/files#diff-6725b21a4aee0ff038410e9e982b9b4c72b64eeae866bdf20768111b16f17a2aR9 for an example,
for counting only at the root you can do just
```suggestion
		countDownloads: `path:"mars5_ar.safetensors"` },
```